### ISSUE TITLE
Black Friday - no prices

### ISSUE THREAD

**Issue by KovalevArtem** [ProtonVPN/proton-vpn-settings#338]
https://protonvpn.com/blackfriday
![image](https://user-images.githubusercontent.com/36500228/99255461-048a3700-2825-11eb-9b48-a25072f09bc0.png)

**Comment by dhoko** [comment]
@KovalevArtem :wave: 
Thx, for the report. Could you give us more details about the browser, env etc.

Do you have JavaScript enabled? because we need it to display this app.
Do you use µMatrix?  If so ensure you allow protonvpn.ch as we load data from there.

**Comment by nico3333fr** [comment]
@dhoko @KovalevArtem Yes, I had the same - I'm using µMatrix. When I enabled the domains, everything went fine.

**Comment by KovalevArtem** [comment]
I have tried different browsers.  On Windows: Chrome and Firefox.  
And also Safari on iPadOS.
![image](https://user-images.githubusercontent.com/36500228/99261225-76667e80-282d-11eb-8a18-5d91d9351368.png)
Prices are not displayed anywhere ...

**Comment by KovalevArtem** [comment]
At the same time on the ProtonMail page everything works everywhere without problems!
![image](https://user-images.githubusercontent.com/36500228/99261961-7024d200-282e-11eb-994a-4352dac27bfb.png)
